src/iuse_actor.h

```diff
@@ -85,6 +88,28 @@ class iuse_transform : public iuse_actor
         void finalize( const itype_id &my_item_type ) override;
 };
 
+class countdown_actor : public iuse_actor
+{
+    public:
+        countdown_actor( const std::string &type = "countdown" ) : iuse_actor( type ) {}
+
+        /** if specified overrides default action name */
+        std::string name;
+
+        /** turns before countdown action (defaults to @ref itype::countdown_interval) */
+        int interval = 0;
```

Comment:
There's a sanity check within the class to ensure values <0 are not used. Changing to `unsigned` does have a habit of causing massive compilation problems.